chapter2.adoc

```diff
@@ -54,6 +54,12 @@ S-interrupt file or an APLIC domain to a Supervisor Domain. The interface also
 describes CSRs to allow M-mode software to retain control on notification of
 interrupts when Supervisor domains are enabled.
 
+* `Smsdedbg` (<<Smsdedbg>>) - This extension defines the external debug
+configuration interface that is managed by M-mode software to declare that
+external debug is allowed for a supervisor domain. Whether external debug is
+authorized or not is expected to be done via a RoT and is outside the scope
+of this specification.
+
```

Comment:
Since the extension is Sm it is already stated that its M-mode.

`Smsdedbg` (<<Smsdedbg>>) - This extension provides the controls to
indicate if external debug is allowed for a supervisor domain.